includes/jsdev/globals.js

```diff
@@ -0,0 +1,9 @@
+requirejs.config({
```

Comment:
Can you add some hints on how this works. Not everybody is familiar with require js.